test/e2e/up.sh

```diff
@@ -27,7 +27,7 @@ curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.25.2/minik
 
 echo "starting minikube..."
 # Using sync-frequency=5s helps to speed up the tests (during the cleanup of resources inside a namespace)
```

Comment:
we can probably reword this to "Using lower value for sync-frequency helps..." so that we don't have to keep this up-to-date with the value we set.(currently they are not in sync 1s vs 5s)

Reply:
done